cmd/jaeger/internal/all-in-one.yaml

```diff
@@ -33,6 +33,11 @@ extensions:
     #   initial_sampling_probability: 0.1
     http:
     grpc:
+  
+  healthcheckv2:
+    use_v2: true
```

Comment:
is this temporary flag or something? Weird that with v2 in the name they have yet another use_v2 flag.

Reply:
Yep, it is a temporary flag. According to them the V2 flag is absolutely necessary while V1 is slowly phased out by them.